packages/components/app/styles/components/accordion.scss

```diff
@@ -0,0 +1,122 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+//
+// ACCORDION
+//
+
+@use "../mixins/button" as *;
+@use "../mixins/focus-ring" as *;
+
+$accordion-item-padding: 16px;
+$accordion-item-medium-border-radius: 6px;
+$accordion-gap: 12px;
+
+// ACCORDION COMPONENT (wrapper)
+
+.hds-accordion {
+  display: flex;
+  flex-direction: column;
+  gap: $accordion-gap;
+}
+
+// ACCORDION ITEM COMPONENT (nested child)
+
+.hds-accordion-item {
+  background: var(--token-color-surface-primary);
+  border-radius: $accordion-item-medium-border-radius;
+
+  &.hds-accordion-item--does-not-contain-interactive {
+    box-shadow: var(--token-surface-mid-box-shadow);
+
+    &:hover,
+    &.mock-hover {
+      box-shadow: var(--token-surface-high-box-shadow);
+    }
+  }
+
+  &.hds-accordion-item--contains-interactive {
+    border: 1px solid var(--token-color-border-primary);
```

Comment:
there is still an issue with the border. I'll show you in a call (its easy to fix).
essentially, you should add a transparent border in any case, and here change only the color to primary

Reply:
I want to wait on changes related to this until we can get more feedback from Christian. 

(I experimented with making changes to the border, padding, and adding a margin as we discussed but it was getting very complex and still didn't seem quite right.)

Reply:
@KristinLBradley visually the accordion items look correct, as you said. I build an accordion using the showcase and the extra pixels add up when there are 3, 5, 6 ... items, so that difference matters. I'd rather reduce the padding or margin if possible.

Other than that, I don't have any more feedback. I reviewed everything again and everything works and looks pretty good.

Reply:
We can pair and try the different options you experimented with directly on the browser if you want. Maybe it's easier or faster to try and review.

Reply:
@cveigt Updated the one which had a border to now use --token-surface-base-box-shadow as you messaged. I compared to Figma and the heights and spacing now look good to me but please doublecheck.